modules/proc_use-api.adoc

```diff
@@ -104,6 +101,77 @@ Use the following procedure to create an OAuth access token.
 This is the only opportunity to copy and save the access token. It cannot be reobtained after leaving this page. 
 ====
 
+[id="reassigning-oauth-access-token"]
+== Reassigning an OAuth access token
+
+{productname} superusers of a registry can take control of API tokens created by other users and re-assign those tokens to a different user. This helps audit logs properly reflect which member of an organization used, or is using, a token after reassignment.
+
+.Prerequisites
+
+* You have created and assigned an OAuth token to a member in your organization.
+* You have superuser administrative privileges.
+
+.Procedure
+
+. Update your {productname} `config.yaml` file to include the `FEATURE_ASSIGN_OAUTH_TOKEN: true` field:
++
+[source,yaml]
+----
+# ...
+FEATURE_ASSIGN_OAUTH_TOKEN: true
+# ...
+----
+
+. Restart your {productname} registry.
+
+. Log in to your {productname} registry as a superuser.
```

Comment:
same, superuser -> organization admin

Reply:
also, this feature only works on Current UI, not on New UI, suggest to note this